readme.md

```diff
@@ -22,7 +22,7 @@ pathIsAbsolute('/home/foo');
 //=> true
 
 // Windows
-pathIsAbsolute('C:/Users/');
+pathIsAbsolute.win32('C:/Users/');
```

Comment:
I think you're missing the point (which might not be that clear). This is intended to be how it work **on** Windows, as with the above for Linux. I would rather clarify the comment.